bin/tennicam_client_display.py

```diff
@@ -21,28 +29,65 @@
 TENNICAM_CLIENT_DEFAULT_SEGMENT_ID = "tennicam_client"
 
 
-def _get_handle():
+def _get_vicon_table_pose(vicon_segment_id: str) -> Transformation:
+    # only import if actually needed
+    import pam_vicon_o80
+
+    logging.info("Get table pose from Vicon (using segment id '%s')", vicon_segment_id)
+    vicon = pam_vicon_o80.PamVicon(vicon_segment_id)
+    vicon.update()
+    return vicon.get_table_pose(yaw_only=True)
+
+
+def _get_handle(vicon_segment_id: t.Optional[str] = None) -> pam_mujoco.MujocoHandle:
     ball = pam_mujoco.MujocoItem(
         "ball", control=pam_mujoco.MujocoItem.CONSTANT_CONTROL, color=(1, 0, 0, 1)
     )
     table = pam_mujoco.MujocoTable("table")
+
+    if vicon_segment_id:
+        table_pose = _get_vicon_table_pose(vicon_segment_id)
+        table.position = table_pose.translation.tolist()
+        table.orientation = table_pose.rotation
+
     graphics = True
     accelerated_time = False
     handle = pam_mujoco.MujocoHandle(
         "tennicam_client_display",
         table=table,
-        balls=(ball,),
+        balls=[ball],
```

Comment:
why replacing a tuple by a list ?

Reply:
mypy was complaining because type hints of `MujocoHandle` explicitly expect a list.  A more proper solution would probably be to change it there to accept a more generic type (e.g. Sequence).